lib/select.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'menu.rb'
+
+class Select
+
+	attr_reader :order, :menu, :due
+
+	def initialize
+		@menu 	= Menu.new
+		@menu 	= @menu.menu
+		@order 	= {}
+		@total 	= []
+		@due 	= 0
+	end
+	def see
+		@menu
+	end
+    
+    def pick(dish , quantity=1)
+    	string_support(dish)
+    	raise 'this dish is not in the menu' unless @menu.include? dish
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)